rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/ApplicationResource.java

```diff
@@ -710,13 +711,7 @@ private void checkLocationsAreValid(ApplicationSpec applicationSpec) {
             return result;
         }
         for (Entity e: descs) {
-            Object v = null;
-            try {
-                v = e.getAttribute((AttributeSensor<?>)s);
-            } catch (Exception exc) {
-                Exceptions.propagateIfFatal(exc);
-                log.warn("Error retrieving sensor "+s+" for "+e+" (ignoring): "+exc);
-            }
```

Comment:
This code has been moved into `EntityAttributesUtils` and re-used in all explicit attempts to retrieve a sensor in `rest-resources` project, see next line.